torch/csrc/jit/tensorexpr/loopnest.cpp

```diff
@@ -647,26 +647,12 @@ LoopNest::LoopNest(const std::vector<Tensor*>& output_tensors)
 Stmt* LoopNest::lowerToStmt(Tensor* t) {
   Function* f = t->function();
   // TODO: Support multiple-output functions
-  Stmt* body = f->ElementStmt(0);
+  auto* body = f->ElementStmt(0, f->body(0));
```

Comment:
There is no need in passing `body` directly: `var_index` (which is 0 in both places) cannot be different for the `func_var` and for the `func_body` because it comes from the tensor `t`. And since we're passing `var_index` in anyway, we could do `Function::body(var_index)` inside the `ElementStmt`